fix(curriculum): validating café menu html tag (#46581)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f33071498eb2472b87ddee4.md

assert(code.match(/<!DOCTYPE html>/i));
```

> You should have an opening `<html>` tag, and set `lang` attribute to `en`.

Reviewer: ```suggestion
You should have an opening `<html>` tag with a `lang` attribute of `en`.
```
Same
